langkit/compile_context.py

```diff
@@ -1822,7 +1822,8 @@ def ext(self, *args):
         args = [a.lower if isinstance(a, names.Name) else a for a in args]
         if self.extensions_dir:
             ret = path.join(*args)
-            if path.isfile(path.join(self.extensions_dir, ret)):
+            p = path.join(self.extensions_dir, ret)
+            if path.isfile(p) or path.isdir(p):
```

Comment:
Can you update the docstring to explicitely describe that it handles directories? (the commit message could make this explicit too :))